ssl/quic/quic_txp.c

```diff
@@ -1469,7 +1469,7 @@ static int txp_should_try_staging(OSSL_QUIC_TX_PACKETISER *txp,
      * This is not a major concern for clients, since if a client has a 1-RTT EL
      * provisioned the server is guaranteed to also have a 1-RTT EL provisioned.
      *
-     * TODO(QUIC SERVER): Revisit this when server support is added.
+     * TODO(QUIC FUTURE): Revisit this when server support is added.
```

Comment:
This text doesn't make any sense since server support is now added. Is there anything that needs revising here, or can we simply remove this comment?

Reply:
I think thats an outstanding question to answer.  The comment above provides some detail about what the problem is, but the RFC doesn't really provide any guidance on how to handle this corner case, nor am I sure either of what to do about it yet, or when we will get to it. Best thing I could think to do here is change the comment to read "Revisit this when we decide how best to handle this"